data/debadityasingh8972.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Debaditya Singh",
+  "displayStatsPublic": ture,
+  "type": "personal",
+  "bio": "Software Developer Fresher Looking for opportunities to test my skills | Full-stack Web Developer | Open-Source Contributor | Blockchain | Ethereum | DeFi | 3⭐ @LeetCode |"
+  "avatar": "https://github.com/debadityasingh8972.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/debaditya5972?t=JmTh6wWhFzsbRK9q6M-7Lg&s=08",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Connect with me on Github",
+      "url": "https://github.com/debadityasingh8972",
+      "icon": "FaGithub"
+    },
```

Also, remove this trailing comma `,` here.
```suggestion
    }
```